src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs

```diff
@@ -158,7 +158,9 @@ public void SelectSupplierWithContactDetails(string bapiConnString)
                 }
             }
 
+#pragma warning disable CA2208
             if (string.IsNullOrEmpty(selectedSupplier)) throw new ArgumentNullException(nameof(selectedSupplier));
+#pragma warning restore CA2208
```

Comment:
I think it might be better to throw a different exception here as `selectedSupplier` isn't an argument. Is there anything else that would fit (excluding `Exception`)?